crypto/threads_pthread.c

```diff
@@ -17,6 +17,13 @@
 # include <atomic.h>
 #endif
 
+#if defined(__apple_build_version__) && __apple_build_version__ < 6000000
+// OS/X 10.7 and 10.8 had a weird version of clang which has __ATOMIC_ACQUIRE and
+// __ATOMIC_ACQ_REL but which expects only one parameter for __atomic_add_fetch()
+// rather than two
```

Comment:
Please do not use C++ style comments, also follow the multiline comment style as elsewhere.

Reply:
@tom-cosgrove-arm nevermind, I'll update it.